contracts/BlobStorageManager.sol

```diff
@@ -151,9 +160,7 @@ contract BlobStorageManager is Ownable {
             bytes32 chunkKey = keccak256(abi.encode(msg.sender, key, chunkIds[i]));
             storageContract.putBlob{value : cost}(chunkKey, i, sizes[i]);
             keyToChunks[key][chunkIds[i]] = chunkKey;
-            if (chunkIds[i] == keyToChunkNum[key]) {
-                keyToChunkNum[key]++;
-            }
+            keyToTotalSize[key] += sizes[i];
```

Comment:
It would be better to handle size manipulation in only one location for better maintainability.

Reply:
fix